net/dns/resolvconf.go

```diff
@@ -98,24 +106,33 @@ func getResolvconfImpl() resolvconfImpl {
 }
 
 type resolvconfManager struct {
-	impl resolvconfImpl
+	logf              logger.Logf
+	impl              resolvconfImpl
+	workaroundApplied bool
```

Comment:
comment?

Reply:
no mutex because I guess managers are never called concurrently? (no need for comment added; just curious)

Reply:
Managers are not called concurrently, correct. At least, I hope it's correct, otherwise all their implementations are wrong :)